endpoints/openrtb2/auction.go

```diff
@@ -1173,12 +1198,12 @@ func isBidderToValidate(bidder string) bool {
 		return false
 	case openrtb_ext.BidderReservedData:
 		return false
+	case openrtb_ext.BidderReservedGPID:
+		return false
 	case openrtb_ext.BidderReservedPrebid:
 		return false
 	case openrtb_ext.BidderReservedSKAdN:
 		return false
-	case openrtb_ext.BidderReservedBidder:
-		return false
```

Comment:
The bidderparams feature does not make special use of "bidder", so I don't see why it needed to add this reserved bidder name. Perhaps it did in an earlier version? Removed.